internal/server/workflow_vars.go

```diff
@@ -119,6 +126,25 @@ func getSelectedTeam(r *http.Request, loggedInTeamId int, defaultTeamId int, tea
 	return model.Team{}, errors.New("invalid team selection")
 }
 
+func createReassignDeputyDropDownList(allTeams []model.Team, requiredTeamTypes []string, currentSelectedTeam model.Team) []model.Team {
+	//show current team page as first in list
+	teamsToReturn := []model.Team{}
+	for _, m := range requiredTeamTypes {
```

Comment:
This ends up looping through the same information twice so the current team can be first in the list. You could do the same in one loop by _prepending_ the selected team instead: 
```

	for _, tt := range requiredTeamTypes {
                if tt == currentSelectedTeam.Type {
                    teamsToReturn = append([]string{currentSelectedTeam}, teamsToReturn)
                }
		for i, m := range allTeams {
			if m.Type == tt && m.Id != currentSelectedTeam.Id {
				teamsToReturn = append(teamsToReturn, allTeams[i])
			}
		}
	}
```